src/plugins/orangehrmAdminPlugin/Service/LocalizationService.php

```diff
@@ -128,4 +130,53 @@ public function getLanguagesArray(I18NLanguageSearchFilterParams $i18NLanguageSe
         $languages = $this->searchLanguages($i18NLanguageSearchParams);
         return $this->getNormalizerService()->normalizeArray(I18NLanguageModel::class, $languages);
     }
+
+    /**
+     * @param int $languageId
+     * @param array $rows
+     * @return void
+     */
+    public function saveAndUpdateTranslatedStringsFromRows(int $languageId, array $rows)
+    {
+        $i18NTranslations = $this->createTranslatedItemsFromRows($languageId, $rows);
+        $this->getLocalizationDao()->saveAndUpdateTranslatedLangString($i18NTranslations);
+    }
+
+    /**
+     * @param int $languageId
+     * @param array $rows
+     * @return array
+     */
+    protected function createTranslatedItemsFromRows(int $languageId, array $rows): array
+    {
+        $i18NTranslations = [];
+        foreach ($rows as $row) {
+            if (!(isset($row['langStringId']))) {
+                throw new LogicException('langStringId is required attribute');
```

Comment:
```suggestion
                throw new \LogicException('langStringId is required attribute');
```
when you are using classes namespace should consider. you have used it from `SebastianBergmann\Type\LogicException` it's wrong